src/pages/Merchant/Register/Register.js

```diff
@@ -1,78 +1,80 @@
-import React, { useState, useRef } from "react";
+import React, { useState } from "react";
 import { useNavigate } from "react-router-dom";
 
 import Button from "components/Button/Button";
 import InputControl from "components/InputControl/InputControl";
 
+import { validateEmail, validateMobile, validateOtp } from "utils/util";
```

Comment:
is there any function named validateOtp in utils ? if there what that does?

Reply:
that will validate otp using regex to avoid api call with improper value